modules/ai/assets/js/editor/ai-behavior.js

```diff
@@ -146,7 +146,7 @@ export default class AiBehavior extends Marionette.Behavior {
 			return;
 		}
 
-		if ( ! shouldShowPromotionIntroduction() ) {
+		if ( ! shouldShowPromotionIntroduction( sessionStorage ) ) {
```

Comment:
where this `sessionStorage` is coming from?

Reply:
window context